python/lsst/ts/standardscripts/auxtel/prepare_for/co2_cleanup.py

```diff
@@ -0,0 +1,141 @@
+                    type: number
+                    default: 0.0
+                el:
+                    description: Target Elevation in degrees.
+                    type: number
+                    minimum: 0.0
+                    maximum: 90.0
+                    default: 20.0
+                rot_tel:
+                    description: Rotator angle in mount physical coordinates (degrees).
+                    type: number
+                    default: 0.0
+                target_name:
+                    description: Name of the position.
+                    type: string
+                    default: "CO2 cleanup position"
+                wait_dome:
+                    description: Wait for dome to be in sync with the telescope?
+                    type: boolean
+                    default: false
+                slew_timeout:
+                    description: Timeout for slew procedure (in seconds).
+                    type: number
+                    default: 180.0
+                ignore:
+                    description: ATCS components to ignore in availability check.
+                    type: array
+                    items:
+                        type: string
+            additionalProperties: false
+        """
+        return yaml.safe_load(schema_yaml)
+
+    async def configure(self, config):
+        """Configure script.
+
+        Parameters
+        ----------
+        config : `types.SimpleNamespace`
+            Script configuration, as defined by `schema`.
+        """
+        self.config = config
+
+        if self.atcs is None:
+            self.atcs = ATCS(
+                domain=self.domain, intended_usage=ATCSUsages.Slew, log=self.log
+            )
+            await self.atcs.start_task
+
+        if hasattr(self.config, "ignore"):
+            for comp in self.config.ignore:
+                if comp not in self.atcs.components_attr:
+                    self.log.warning(
+                        f"Component {comp} not in ATCS. "
+                        f"Must be one of {self.atcs.components_attr}. Ignoring."
+                    )
+                else:
+                    self.log.debug(f"Ignoring component {comp}.")
+                    setattr(self.atcs.check, comp, False)
+
+    def set_metadata(self, metadata):
+        metadata.duration = self.slew_time_guess
+
+    async def run(self):
+        await self.atcs.assert_all_enabled()
+
+        self.log.debug(
+            "Slew telescope to CO2 cleanup position. "
+            f"Az: {self.config.az}, "
+            f"El: {self.config.el}, "
+            f"Rot: {self.config.rot_tel}"
+        )
+        await self.atcs.point_azel(
+            az=self.config.az,
+            el=self.config.el,
+            rot_tel=self.config.rot_tel,
+            target_name=self.config.target_name,
+            wait_dome=self.config.wait_dome,
+            slew_timeout=self.config.slew_timeout,
+        )
```

Comment:
After moving the telescope you should disable the ATAOS corrections and open the mirror covers for cleaning, e.g.

```
await atcs.disable_ataos_corrections()
await atcs.open_m1_cover()
```

Reply:
I see that ``atcs.open_m1_cover`` requires a minimum elevation of 70 degrees, so I think it would be best to change the default elevation in the configuration to this value. Is that ok?

Reply:
Right! We should probably open m1 before moving then...

Reply:
Is it ok to move the telescope with the mirror cover open?

Reply:
Yes